### ISSUE TITLE
(Neo)Vim - Display gets corrupted from search/scroll navigations.

### ISSUE THREAD

**Issue by zelhar** [termux/termux-packages#1418]
This is a problem which occurs when the terminal window is not considerably zoomed out  to the point that the font is too small.

To reproduce this bug:
open a long document in vim/neovim. search a term and scroll through the results. sometimes just scrolling down produces similar corruption. What happens is that some lines get reproduced or broken including the status bar. The cursor overshoots the found result and is displayed bellow  and basically the document is unreadable until you either redraw! (Ctrl-l) or use pinch zoom to zoom out. 

Now I am not talking about documents with very long lines of text, it happens with documents with textwidth of 80 or less with plenty of room in the terminal screen. 

When I zoom out the terminal to some smaller font size, the problem goes away except the font is really to small.

**Comment by fornwall** [comment]
Thanks for reporting!

Could you give some more exact steps for reproducing? Perhaps by sharing your exact terminal size (run ´pkg install vutils`, then see the terminal size by running `vtsize`) before running vim, and sharing a text document where you can easily reproduce the problem?

**Comment by zelhar** [comment]
Hello and thanks for replying. I believe I know the cause and it is probably only vim related and so my apology for bothering you.

to reproduce the problem
I set sessionoptions to:
ˋset sessionoptions=blank,buffers,curdir,folds,resize,slash,unix,winpos,winsize,tabpagesˋ
I autoload the session on vim startup. 
when I resize (zoom in) termux, then start vim again, it gets messed up probably because it tries to fit its old size into a too small terminal screen.

When I removed the ˋresize,winpos,winsizeˋ  from sessionoptions the problem goes away. I haven't yet narrowed down which of the 3 is the actual offender.

Thanks again and sorry for falsely bugreporting.
